jbpm-designer-backend/src/main/java/org/jbpm/designer/repository/vfs/VFSRepository.java

```diff
@@ -666,727 +682,98 @@ public void createGlobalDirOnNewProject(@Observes NewProjectEvent newProjectEven
-                                    "                \"parent\" : [\"2\"],\n" +
-                                    "                \"children\" : [\"5\"],\n" +
-                                    "                \"xyOffset\" : [60,-45]\n" +
-                                    "            },\n" +
-                                    "            {\n" +
-                                    "                \"id\" : \"4\",\n" +
-                                    "                \"name\": \"c\",\n" +
-                                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#IntermediateMessageEventCatching\",\n" +
-                                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                                    "                \"connectingType\": \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                                    "                \"parent\" : [\"2\"],\n" +
-                                    "                \"children\" : [\"6\"],\n" +
-                                    "                \"xyOffset\" : [60,45]\n" +
-                                    "            },\n" +
-                                    "            {\n" +
-                                    "                \"id\": \"5\",\n" +
-                                    "                \"name\": \"B\",\n" +
-                                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#Task\",\n" +
-                                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                                    "                \"connectingType\" : \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                                    "                \"parent\" : [\"3\"],\n" +
-                                    "                \"children\" : [],\n" +
-                                    "                \"xyOffset\" : [120,0]\n" +
-                                    "            },\n" +
-                                    "            {\n" +
-                                    "                \"id\": \"6\",\n" +
-                                    "                \"name\": \"C\",\n" +
-                                    "                \"nodetype\" : \"http://b3mn.org/stencilset/bpmn2.0#Task\",\n" +
-                                    "                \"namespace\" : \"http://b3mn.org/stencilset/bpmn2.0#\",\n" +
-                                    "                \"connectingType\" : \"http://b3mn.org/stencilset/bpmn2.0#SequenceFlow\",\n" +
-                                    "                \"parent\" : [\"4\"],\n" +
-                                    "                \"children\" : [],\n" +
-                                    "                \"xyOffset\" : [120,0]\n" 
```

Comment:
Hi @tsurdilo I believe this could be replaced be this method https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html#readAllBytes(java.nio.file.Path)

In our specific case:
`Files.readAllBytes(file.toPath())`